src/dagre-js/create-clusters.js

```diff
@@ -2,9 +2,9 @@ import * as d3 from "d3";
 import { addLabel } from "./label/add-label";
 import * as util from "./util";
 
-export { createClusters };
+export { createClusters, setCreateClusters };
 
-function createClusters(selection, g) {
+var createClusters = function (selection, g) {
```

Comment:
Do you mind using `const` instead of `var`?

Reply:
`const` is part of ES6 specification. Assuming that package is compliant with all browser versions and compared to the current codebase. `var` seems more appropriate. What do you think?

Reply:
What's more, `createClusters` is not a constant but a variable. So maybe use `let` at worst